ucp/core.py

```diff
@@ -405,10 +384,11 @@ def port(self):
 
     def close(self):
         """Closing the listener"""
-        if not self._closed:
-            self._b.abort()
-            self._closed = True
-            self._b = None
+        self._b.close()
+
+    def abort(self):
+        """Closing the listener"""
+        self._b.close()
```

Comment:
Why do we need both `abort` and `close` if they do the same thing?

Reply:
Removed :)
(Used `abort()` in a intermediate develop step)